### ISSUE TITLE
Annihilation Plane unexpectedly destroys drops when breaking certain blocks.

### ISSUE THREAD

**Issue by Birgliss** [AppliedEnergistics/Applied-Energistics-2#4931]
**Describe the bug**
Discovered while trying to automate the Orechid Ignem from Botania.
While most blocks work fine, breaking Nether Quartz Ore with an Annihilation Plane consistently does not yield any items.
I have also noticed this occur with blocks from other mods, notably Aluminium Ore from ZYcraft. I do not know if it effects any non-ore blocks.

**To Reproduce**
Break Nether Quartz Ore with an Annihilation Plane or Identity Annihilation Plane.
(The bug does not occur when picking up items with an Annihilation Plane. It only occurs with placed blocks.)

**Expected behavior**
Nether Quartz (or Nether Quartz Ore) should enter the system, as expected with any other block.

**Additional context**
Here is a screenshot of the setup I used to reproduce the bug.
There is an item import bus underneath the skystone chest on the left.
The left-chest is filled with Nether Quartz ore.
The right-chest is attached to a storage bus and is where the expected output should go. It is unfiltered.
I reproduced this in singleplayer.
![2021-01-11_00 34 20](https://user-images.githubusercontent.com/20749209/104149639-05fb5a00-53a5-11eb-9602-fe339709a55d.png)

**Environment**

[Forced crash report (debug)](https://gist.github.com/Birgliss/33e1c5147e04dea3c5cc58a801357bb1)
- Minecraft: 1.16.4
- Forge: 35.1.36
- AE2: 8.2.0-beta2

**Comment by yueh** [comment]
Duplicate of #4692

**Comment by Birgliss** [comment]
I'm confused. Both of these issue reports have been closed, but neither time has the bug been fixed nor has there been an explanation for why it hasn't.
Do you have a plan to fix this? Are we waiting on Forge?

**Comment by yueh** [comment]
It's up to forge to fix it.

**Comment by mindforger** [comment]
> 
> 
> We use forge to obtain the correct tool for a block, which in many cases is not maintained at all by them.
> 
> Hopefully they fix it or provide a working alternative, but for now there is not really something we can do about it.

so your second guess is in the right direction but it is in the ballpark of the mods to declare the right utility to break the block via Forge API

**Comment by mindforger** [comment]
i misread your other answer that is an issue with the mod not maintaining their contrat to tell which tool to use ?!
